app/src/main/res/layout/fragment_tab_tray.xml

```diff
@@ -0,0 +1,98 @@
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        mozac:tabsTrayItemTextColor="?tabTrayItemText"
+        mozac:tabsTraySelectedItemTextColor="?tabTrayItemSelectedText"
+        mozac:tabsTrayItemBackgroundColor="?tabTrayItemBackground"
+        mozac:tabsTraySelectedItemBackgroundColor="?tabTrayItemSelectedBackground" />
+
+    <LinearLayout
+        android:id="@+id/saveToCollection"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:paddingStart="16dp"
+        android:paddingEnd="16dp"
+        app:layout_constraintTop_toBottomOf="@+id/tabsTray"
+        app:layout_constraintBottom_toTopOf="@id/bottomBarShadow">
+        <include
+            layout="@layout/save_to_collection_button"
+            android:id="@+id/saveToCollectionButton"/>
+    </LinearLayout>
+
+    <View
+        android:id="@+id/bottomBarShadow"
+        android:layout_width="0dp"
+        android:layout_height="1dp"
+        android:background="@color/bottom_bar_shadow"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintBottom_toTopOf="@id/tab_tray_controls" />
+
+    <LinearLayout
+        android:id="@+id/tab_tray_controls"
+        android:layout_height="48dp"
+        android:layout_width="0dp"
+        android:orientation="horizontal"
+        android:gravity="center"
+        android:background="?bottomBarBackground"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintBottom_toBottomOf="parent">
+        <ImageButton
+            android:id="@+id/private_browsing_button"
+            android:layout_width="@dimen/glyph_button_height"
+            android:layout_height="@dimen/glyph_button_height"
+            android:background="?android:attr/selectableItemBackgroundBorderless"
+            android:contentDescription="@string/content_description_private_browsing_button"
+            android:src="@drawable/private_browsing_button"
+            android:layout_weight=".3333" />
+        <ImageButton
+            android:id="@+id/tab_tray_go_home"
+            android:layout_width="@dimen/glyph_button_height"
+            android:layout_height="@dimen/glyph_button_height"
+            android:background="?android:attr/selectableItemBackgroundBorderless"
+            android:contentDescription="FIX ME: Home"
+            android:src="@drawable/ic_home"
+            android:layout_weight=".3333" />
+        <ImageButton
+            android:id="@+id/tab_tray_open_new_tab"
+            android:layout_width="@dimen/glyph_button_height"
+            android:layout_height="@dimen/glyph_button_height"
+            android:background="?android:attr/selectableItemBackgroundBorderless"
+            android:contentDescription="FIX ME: NEW TAB"
```

Comment:
Are we landing with these content descriptions? You can make string resources for content descriptions with what you think makes sense.